sdl2promela/translator.py

```diff
@@ -473,9 +495,35 @@ def __generate_expression(context: Context, expression: sdlmodel.ProcedureCall):
     # into separate expressions for the calls, assigning results to intermediate values
     # and then using this values in the actual expression
     # TODO
-    raise NotImplementedError(
-        f"Call to {str(expression.name)} cannot be made within an expression"
-    )
+
+    if expression.name.lower() == "present":
+        # A special case when context shall be changed
+        if len(expression.parameters) != 1:
```

Comment:
THis could also benefit from the parameter check extraction

Reply:
I prefer to not use private function from builtins.py